### ISSUE TITLE
Add browsable link to the features database

### ISSUE THREAD

**Issue by jkosinski** [KosinskiLab/AlphaPulldown#466]
At the moment, it is not possible to easily see which organisms are available in the features database:
https://github.com/KosinskiLab/AlphaPulldown/blob/240551bdf0dc81edd880781094b8886148567cd6/README.md?plain=1#L1802

I think there was a browsable link that one could check with the browser. Could you add it? It would be important for the submission.

Also, this database is hard to find on this page, not sure how to do it better, maybe some quick links at the very top of the page? Also important for the submission so reviewers can easily find it.

**Comment by jkosinski** [comment]
Hmm, I cannot see the link in the readme?

**Comment by DimaMolod** [comment]
yes, I removed it for the time being, but will uncomment back as soon as the corrected features are uploaded. The s3 server and html scripts are up and running
